src/cli/commands/fellowship/startGame.ts

```diff
@@ -0,0 +1,158 @@
+import { Command } from "commander";
+import chalk from "chalk";
+import ora from "ora";
+import { createPublic, createWallet } from "../../client";
+import RankifyPlayer from "../../../rankify/Player";
+import { resolvePk } from "../../getPk";
+import GameMaster from "../../../rankify/GameMaster";
+import { CLIUtils } from "../../utils";
+import InstanceBase from "../../../rankify/InstanceBase";
+import { gameStatusEnum } from "../../../types";
+import { BlockchainUtils } from "../../../utils/blockchain";
+
+export const startGame = new Command("startGame")
+  .description("Start a game in a Rankify instance")
+  .argument("<instance>", "Address or index of the Rankify instance")
+  .argument("<gameId>", "ID of the game to start")
+  .option("-r, --rpc <url>", "RPC endpoint URL. If not provided, RPC_URL environment variable will be used")
+  .option(
+    "-k, --key <privateKey>",
+    "Private key or index to derive from mnemonic for signing transactions. If not provided, PRIVATE_KEY environment variable will be used"
+  )
+  .option(
```

Comment:
I would prefer it takes signature here, so that it can be used as CLI on client side as well